experimental/FTheoryTools/docs/src/literature.md

```diff
@@ -120,3 +120,80 @@ Provided that a resolution for a model is known, we can (attempt to) resolve the
 ```@docs
 resolve(m::AbstractFTheoryModel, index::Int)
 ```
+
+
+## The Quadrillion F-Theory Standard Models
+
+A yet more special instance of literature models are the Quadrillion F-theory Standard Models
+(F-theory QSMs) [CHLLT19](@cite). Those hypersurface models come in 708 different families.
+
+The base geometry of an F-theory QSM is obtained from triangulating one of 708 reflexive 3-dimensional
+polytopes. The models, whose bases are obtained from triangulations of the same polytope form a family.
+The following information on the polytope in question and its triangulations is available within our database:
+```@docs
+vertices(m::AbstractFTheoryModel)
+polytope_index(m::AbstractFTheoryModel)
+has_quick_triangulation(m::AbstractFTheoryModel)
+max_lattice_pts_in_facet(m::AbstractFTheoryModel)
+estimated_number_of_triangulations(m::AbstractFTheoryModel)
+```
+
+Beyond the polytope and its triangulations, a number of other integers are of key importance. The following
+are supported in our database.
+```@docs
+kbar3(m::AbstractFTheoryModel)
+hodge_h11(m::AbstractFTheoryModel)
+hodge_h12(m::AbstractFTheoryModel)
+hodge_h13(m::AbstractFTheoryModel)
+hodge_h22(m::AbstractFTheoryModel)
+```
+
+More recently, a research program estimated the exact massless spectra of the F-theory QSMs
+(cf. [Bie24](@cite)). These studies require yet more information about the F-theory QSM geometries,
+which are supported by our database.
+
+First, recall that (currently), the base of an F-theory QSM is a 3-dimensional toric variety B3.
+Let s in H^0(B3, Kbar_B3), then V(s) is a K3-surface. Moreover, let xi be the coordinates
+of the Cox ring of B3. Then V(xi) is a divisor in B3. Consequently, Ci = V(xi) cap V(s)
+is a divisor in the K3-surface V(s). For the root bundle counting program, these curves Ci are
+of ample importance (cf. [Bie24](@cite)). We support the following information on these curves:
+```@docs
+genera_of_ci_curves(m::AbstractFTheoryModel)
+degrees_of_kbar_restrictions_to_ci_curves(m::AbstractFTheoryModel)
+topological_intersection_numbers_among_ci_curves(m::AbstractFTheoryModel)
+indices_of_trivial_ci_curves(m::AbstractFTheoryModel)
+topological_intersection_numbers_among_non_trivial_ci_curves(m::AbstractFTheoryModel)
```

Comment:
I would probably replace `_non_trivial_` here with `_nontrivial_`. Both "non-trivial" and "nontrivial" are in common use, and this function name already has so many underscores in it

Reply:
Ok. Let me adjust this...

Reply:
Just updated the PR accordingly.

Reply:
Third attempt now. The doc tests just keep failing repeatedly due to printing changes in dicts. :(